os_migrate/roles/conversion_host/tasks/conv_hosts_inventory.yml

```diff
@@ -26,3 +26,19 @@
       os_migrate_conversion_ca_cert: "{{ os_migrate_dst_ca_cert|default(omit) }}"
```

Comment:
I'd add another key, e.g.:

```
inventory_group: conversion_hosts_dst
```

Reply:
sure thing, I wasnt sure if you wanted the extra key

Reply:
done

Reply:
Thanks, re-adding the e2e CI labels.